data.md

```diff
@@ -337,23 +347,84 @@ One needs to unname the `ValTypes` first before calling the `#take` or `#drop` f
 Strings
 -------
 
+Wasm use a different character escape rule with K, so we need to define the `unescape` function ourselves.
+
+```k
+    syntax String ::= unescape(String)              [function]
+                    | unescape(String, Int, String) [function, klabel(unescapeAux)]
+ // -------------------------------------------------------------------------------
+    rule unescape(S) => unescape(S, 1, "")
+    rule unescape(S, IDX, SB) => unescape(S, IDX +Int 1, SB +String substrString(S, IDX, IDX +Int 1))
+      requires IDX <Int lengthString(S) -Int 1
+       andBool substrString(S, IDX, IDX +Int 1) =/=K "\\"
```

Comment:
... an explanation here that escaped digrams of hex characters (e.g., "\5F"), get converted to their corresponding byte, and then to the character for that byte (so "\5F" => "_") ...